.github/actions/update-mod-version/update-mod.sh

```diff
@@ -4,21 +4,28 @@ set -euo pipefail
 branch=$1
 semvar_bump=$2
 
+# echo "Remote is - $(git remote -v)"
+git_remote=$(git config --get remote.origin.url)
+searchstring="https://github.com/"
+git_remote=${git_remote#*$searchstring}
+git_remote=${git_remote%".git"}
+echo "$git_remote"
+
 git config user.name github-actions
 git config user.email github-actions@github.com
 git config --global url."https://$GITHUB_TOKEN:x-oauth-basic@github.com/".insteadOf "https://github.com/"
```

Comment:
This is pretty cool. How does it work? Does it only work on GitHub hosted runners?

Reply:
I have re-jigged this now that i'm doing the checkout of the target repo in the script. Now I just split on the first / and remove the first part of the url from the target-repo param. This makes it more generic as it's based on the url string that is set.